app/src/renderer/indexed_model.rs

```diff
@@ -13,28 +13,35 @@ pub struct IndexedModel {
     pub tex_coord_indices: Vec<u16>
 }
 
+pub const BYTE_SIZE_F32: u8 = 4;
+pub const SIZE_POSITION: u8 = 3;
+pub const BYTE_SIZE_POSITION: u8 = BYTE_SIZE_F32 * SIZE_POSITION;
+pub const SIZE_TEX_COORD: u8 = 2;
+pub const BYTE_SIZE_TEX_COORD: u8 = BYTE_SIZE_F32 * SIZE_TEX_COORD;
```

Comment:
`core::mem::size_of::<f32>()` which I think is const could replace this and make it be able to change stuff
https://doc.rust-lang.org/core/mem/fn.size_of.html

Reply:
Thanks! Applied your suggestion, this is much better :)